packages/cactus-test-tooling/src/main/typescript/fabric/fabric-samples-env-constants.ts

```diff
@@ -0,0 +1,125 @@
+export interface IFabricOrgEnvInfo extends NodeJS.ProcessEnv {
+  [key: string]: string | undefined;
+  readonly CORE_PEER_TLS_ENABLED: string;
+  readonly CORE_PEER_LOCALMSPID: string;
+  readonly CORE_PEER_TLS_CERT_FILE: string;
+  readonly CORE_PEER_TLS_KEY_FILE: string;
+  readonly CORE_PEER_ADDRESS: string;
+  readonly CORE_PEER_MSPCONFIGPATH: string;
+  readonly CORE_PEER_TLS_ROOTCERT_FILE: string;
+  readonly ORDERER_TLS_ROOTCERT_FILE: string;
+}
+
+export const FABRIC_25_LTS_FABRIC_SAMPLES__CORE_PEER_TLS_ENABLED_ORG_1 = "true";
```

Comment:
Can't we add a helper method in fabric-test-ledger-v1 class for reading env variables from container and use them instead? We could also add a method that would return ready configs for different orgs (i.e. multiple variables needed for by gateway). That way we're not hardcoding paths in code and are safe in case some of them changes in the future (very unlikely, hence this is just an optional comment, but still to consider).